openapi.go

```diff
@@ -338,18 +338,45 @@ func (i *Items) ProtoMessage(msgName, name string, defs map[string]*Items, indx
 func protoComplex(i *Items, typ, msgName, name string, defs map[string]*Items, index *int, depth int) string {
 	switch typ {
 	case "object":
+		// make a map of the additional props we might get
+		addlProps := make(map[string]string)
```

Comment:
Tiny style thing, sorry.

At NYT we prefer to reserve `make` for channels and allocate slices. Mind switching this to `map[string]string{}`?